### PR TITLE
resource/alicloud_cs_managed_kubernetes: add param delete_options for delete operation; resource/alicloud_cs_kubernetes: add param delete_options for delete operation.

### PR DESCRIPTION
**Pull request by sanyangji**

### CODE REVIEW

File: alicloud/resource_alicloud_cs_kubernetes_test.go [hunk 1]
```diff
@@ -349,6 +349,30 @@ func TestAccAliCloudCSKubernetes_basic(t *testing.T) {
...
+							"resource_type": "SLS_ControlPlane",
+						},
+						{
+							"delete_mode":   "delete",
+							"resource_type": "ALB",
+						},
+					},
+				}),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheck(map[string]string{})),
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_cs_kubernetes_test.go:374]: 要在增加一个只保留一个 delete mode 的step

File: alicloud/resource_alicloud_cs_kubernetes.go [hunk 2]
```diff
@@ -1931,7 +1959,7 @@ func fetchWorkerNodes(d *schema.ResourceData, meta interface{}) []map[string]int
 	var response *roacs.DescribeClusterNodePoolsResponse
 	wait := incrementalWait(3*time.Second, 3*time.Second)
 	err = resource.Retry(5*time.Minute, func() *resource.RetryError {
-		response, err = csClient.DescribeClusterNodePools(tea.String(d.Id()))
+		response, err = csClient.DescribeClusterNodePools(tea.String(d.Id()), &roacs.DescribeClusterNodePoolsRequest{})
 		if err != nil {
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_cs_kubernetes.go:1963]: read 里面 set 的逻辑呢？

**Review by xiaozhu36 — CHANGES_REQUESTED** [review]

**Review by xiaozhu36 — APPROVED** [review]

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Pull request merged**